src/cli.ts

```diff
@@ -88,6 +91,16 @@ const freezeOpt: opt = [
   { default: false },
 ];
 const silentOpt: opt = ["--silent", "Silence all output", { default: false }];
+const buildOpt: opt = [
+  "--entry <file>",
```

Comment:
Instead of making this a flag, we should just make these the arguments to the build. Also because Rollup supports multiple entry points, we should support multiple arguments too.